guide/src/debugging.md

```diff
@@ -16,7 +16,7 @@ You can also debug classic `!`-macros by adding `-Z trace-macros`:
 cargo rustc --profile=check -- -Z unstable-options --pretty=expanded -Z trace-macros > expanded.rs; rustfmt expanded.rs
 ```
 
-See [cargo expand](https://github.com/dtolnay/cargo-expand) for a more elaborate version of those commands.
+Note that those commands require using the nightly build of rust and are prone to errors. See [cargo expand](https://github.com/dtolnay/cargo-expand) for a more elaborate and stable version of those commands.
```

Comment:
Thanks! I see what you're going for with this point, though for some reason the choice of words "and are prone to errors" feels a little off, I'm struggling to say exactly why though.

What do you think of something like this:

```suggestion
Note that those commands require using the nightly build of rust and may occasionally have bugs. See [cargo expand](https://github.com/dtolnay/cargo-expand) for a more elaborate and stable version of those commands.
```

Reply:
It seems fine, I just wanted to point out that I could not run any of those commands either in nightly nor stable. Mainly to bugs in third party libraries conflicting with experimental features like const_fn or llvm macro. So that if anyone tries and fails, knows it is normal.